docs/UserGuide.md

```diff
@@ -3,91 +3,128 @@
+&nbsp;&nbsp;[1.2 Product Overview](#12-product-overview) <br>
+&nbsp;&nbsp;[1.3 Keywords](#13-keywords)<br>
+[2. **Quick Start**](#2-quick-start)<br>
+[3. **Features**](#3-features) <br>
+&nbsp;&nbsp;[3.1 Viewing help](#31-viewing-help-help) <br>
+&nbsp;&nbsp;[3.1.1 Viewing help for a specific command](#311-viewing-help-for-a-specific-command-help-command_word) <br>
+&nbsp;&nbsp;[3.2 Add an item](#32-adding-an-item-add-opt-args) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.2.1 Add a task](#321-adding-a-task-add--t--by) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.2.2 Add a module](#322-adding-a-module-add--m) <br>
+&nbsp;&nbsp;[3.3 Edit an item](#33-editing-an-item-edit-opt-args) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.3.1 Edit a task description](#331-editing-a-task-description-edit--t)<br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.3.2 Editing a module](#332-editing-a-module-edit--m) <br>
+&nbsp;&nbsp;[3.4 Delete an item](#34-deleting-an-item-del-opt-args) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.4.1 Delete a task](#341-deleting-a-task-del--t) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.4.2 Delete a module](#342-deleting-a-module-del--m) <br>
+&nbsp;&nbsp;[3.5 List items](#35-listing-all-items-list-opt-args) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.5.1 List tasks](#351-listing-all-tasks-list--t) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.5.2 List modules](#352-listing-all-modules-list--m)  <br>
+&nbsp;&nbsp;[3.6 Grade a module](#36-grade-a-existing-module-grade) <br>
+&nbsp;&nbsp;[3.7 Calculate CAP](#37-calculate-your-cap-after-the-semester-cap) <br>
+&nbsp;&nbsp;[3.8 Mark task as done](#38-mark-as-done-done-task_index) <br>
+&nbsp;&nbsp;[3.9 Undo previous command](#39-undo-a-command-undo) <br>
+&nbsp;&nbsp;[3.10 Summary](#310-summary-summary) <br>
+&nbsp;&nbsp;[3.11 Timetable](#311-timetable-timetable-opt-args) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.11.1 View timetable](#3111-view-the-timetable-timetable-args) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.11.2 Add a lesson](#3112-add-a-lesson-timetable--add) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;[3.11.3 Delete a lesson](#3113-delete-a-lesson-timetable--del) <br>
+&nbsp;&nbsp;[3.12 Exit](#312-exiting-the-program-bye) <br>
+[4. **FAQ**](#4-faq) <br>
+[5. **Command Summary**](#5-command-summary) <br>
+
+### 1 Introduction
+
+Repository Assistant with a Versatile Interface (ra.VI) is a desktop app for managing tasks, deadlines, and notes, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ra.VI can manage your tasks faster than traditional GUI apps.\
+ra.VI helps you to manage your tasks, and their corresponding deadlines with ease. It can also help you plan your timetable and being organised keeps you ahead of the curve.
 
 ### 1.1 About
 
 This user guide provides in-depth documentation on the ra.VI installation process, system configuration and management. In addition, the quick start guide provides and end-to-end setup process to begin tracking your tasks and timetable with ra.VI.
```

Comment:
I couldn't find an installation process section in this whole doc?

Reply:
Amended